src/buildblock/GeometryBlocksOnCylindrical.cxx

```diff
@@ -156,11 +165,77 @@ build_crystal_maps()
 	std::string scanner_orientation = get_scanner_ptr()->get_scanner_orientation();
 
 	// check for the scanner orientation
-	if (scanner_orientation=="Y" || num_transaxial_buckets%4==0 )
-	{/*Building starts from a bucket perpendicular to y axis, from its first crystal.
+	/*Building starts from a bucket perpendicular to y axis, from its first crystal.
 		see start_x*/
 
 		//calculate start_point to build the map.
+<<<<<<< HEAD
```

Comment:
well. no. You've now committed the file that contains the conflict, i.e. it has both versions (your code here, and the original `pkhateri/master` code below). You need to manually edit the file.

Reply:
ok figured out what happened. VS code trying to be helpful shows the conflict as an overlay and some options to resolve the conflict. Unfortunately, unless I select an option it will keep both changes which is what it did :) .